### DISCUSSION TITLE
Shouldn't lightgallery REPLACE the images?

### DISCUSSION THREAD

**Discussion by zackrspv** [sachinchoolur/lightGallery#1488] [Q&A]
Shoudln't the lightgallery when implemented and open, replace the actual root images? 

![image](https://user-images.githubusercontent.com/1069198/228335015-4811f9b4-f054-4d0c-8768-e267f046cade.png)

**Comment by sachinchoolur** [comment]
Nope, lightGalley does not make any modifications to the root images. It only opens the images in a lightBox gallery

**Reply by zackrspv** [reply]
IF we are going to display the gallery inline though, wouldn't that be wanting to replace the base image? I mean, the whole purpose of using it inline is to put it in PLACE of the image.
